.github/workflows/iroha2-glibc-image.yml

```
@@ -0,0 +1,48 @@
+name: I2::GLIBC::Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      IROHA2_IMAGE_TAG:
+        required: true
+        default: stable
+      IROHA2_IMAGE_RELEASE:
+        required: true
+      IROHA2_DOCKERFILE:
+        required: true
+        default: Dockerfile.glibc.profiling
+
+jobs:
+  dockerhub:
+    runs-on: [self-hosted, Linux, iroha2-dev-push]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Login to Soramitsu Harbor
+        uses: docker/login-action@v3
+        with:
+          registry: docker.soramitsu.co.jp
+          username: ${{ secrets.HARBOR_USERNAME }}
+          password: ${{ secrets.HARBOR_TOKEN }}
+      - name: Set up Docker Buildx
+        id: buildx
+        if: always()
+        uses: docker/setup-buildx-action@v3
+        with:
+          install: true
+      - name: Build and push iroha2:glibc image
+        uses: docker/build-push-action@v5
+        if: always()
+        with:
+          push: true
+          tags: |
+            hyperledger/iroha2:${{ github.event.inputs.IROHA2_IMAGE_TAG }}-${{ github.event.inputs.IROHA2_IMAGE_RELEASE }}-glibc
+            docker.soramitsu.co.jp/iroha2/iroha2:${{ github.event.inputs.IROHA2_IMAGE_TAG }}-${{ github.event.inputs.IROHA2_IMAGE_RELEASE }}-glibc
```

```suggestion
            hyperledger/iroha2:${{ github.event.inputs.IROHA2_IMAGE_TAG }}-${{ github.event.inputs.IROHA2_IMAGE_RELEASE }}-profiling
            docker.soramitsu.co.jp/iroha2/iroha2:${{ github.event.inputs.IROHA2_IMAGE_TAG }}-${{ github.event.inputs.IROHA2_IMAGE_RELEASE }}-profiling
```

the important thing is that it's an image for profiling, the fact that it uses `glibc` is just an implementation detail

---

I'm not sure what `IROHA2_IMAGE_RELEASE` is. Is it the name of the profile?